FitGoal/FitGoal/Setting/SettingsViewController.swift

```diff
@@ -77,9 +116,27 @@ class SettingsViewController: UITableViewController {
     }
 }
 
+private enum SettingsTableViewSection: Int, CaseIterable {
+    case userInfo
+    case accountInfo
+}
+
 struct UserPreference {
     let preferenceName: String
     let preferenceImage: UIImage
     let preferenceValue: String
 }
 
+enum AccountInfoCellType {
+    case notifications(accessoryView: UIView)
+    case accountInfo(accessoryType: UITableViewCell.AccessoryType)
+    
+    var name: String {
+        switch self {
+        case .notifications:
+            return "Notifications"
+        case .accountInfo:
+            return "Account Info"
+        }
+    }
+}
```

Comment:
So these are the changes that I propose. Downside: It's a new `enum`.  What do you recond? It's almost always the solution that comes to my mind when I face a problem like this.